packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -75,6 +77,25 @@ export abstract class ActorQueryOperation extends Actor<IActionQueryOperation, I
     }
   }
 
+  public static getExpressionContext(context: ActionContext): IExpressionContext {
+    if (context) {
+      const now: Date | undefined = context.get(KEY_CONTEXT_QUERY_TIMESTAMP, undefined);
+      const baseIRI: string | undefined = context.get(KEY_CONTEXT_BASEIRI, undefined);
+      return { now, baseIRI };
+    } else {
+      return {
```

Comment:
Is setting these entries explicitly to undefined necessary? Wouldn't an empty hash work?